src/locale/es.json

```
@@ -30,17 +30,76 @@
     "areYouSure": "¿Estás absolutamente seguro?",
     "updateButton": "Actualizar",
     "updateConfirmation": "¿Estás seguro de que deseas iniciar una actualización de software? Esto puede tomar varios minutos. No podrás iniciar un stream hasta que se haya completado la actualización. El codificador se desconectará brevemente después de una actualización exitosa. Nunca elimines la potencia o resetees el encoder mientras se realiza la actualización. Si el codificador es alimentado por batería, asegúrate de que esté completamente cargado.",
-    "streamingMessage": "Su transmisión está utilizando {usingNetworksCount} redes con una latencia de {srtLatency} ms"
+    "streamingMessage": "Su transmisión está utilizando {usingNetworksCount} redes con una latencia de {srtLatency} ms",
```

Biome (2.1.2)

[error] The key streamingMessage was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)